arquillian-environment-reporter/arquillian-environment-reporter-impl/src/test/java/org/arquillian/environment/reporter/impl/TestRunnerDetectorTest.java

```diff
@@ -0,0 +1,17 @@
+package org.arquillian.environment.reporter.impl;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestRunnerDetectorTest {
+
+    @Test
+    public void should_return_maven_as_runner() {
+
+        final TestRunnerDetector.TestRunner testRunner = TestRunnerDetector.detect();
+
+        assertThat(testRunner).isEqualTo(TestRunnerDetector.TestRunner.MAVEN);
```

Comment:
Yes if you run from IDE fails, but well not sure if it was necessary to fix in IDE or just be compatible with build tool.